test/e2e/lint-gate-verify_test.sh

```
@@ -46,10 +46,28 @@ run_test() {
 
-	# Run the verifier against the fixture
+	# CodeQL default setup auto-detects any go.work as a Go workspace and cannot
+	# be given a paths-ignore. Fixtures are stored as go.work.fixture and staged
+	# in a temp dir at test time, renamed to go.work for verifier execution.
+	local tmp
+	tmp=$(mktemp -d) || {
+		echo -e "${RED}✗${NC} $fixture ($rule): Failed to create temp dir"
+		TESTS_FAILED=$((TESTS_FAILED + 1))
+		return 1
+	}
+	trap "rm -rf '$tmp'" RETURN
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)